AnselemOdims/Loaner
# #165541393 Install and Setup ESlint

### What does this PR do?
Sets up the ESlint for linting of code and extends Airbnb style guide

#### Description of Task to be completed?
- Install ESLint locally as a dev-dependency using "npm i --save-dev eslint"
- Extend Airbnb by installing it with "npm i --save-dev eslint-config-airbnb"
- Configure it with a .eslintrc file 
- Add the import plugin with "npm i --save-dev eslint-plugin-import"

#### How can it be manually tested
- Clone this repository
- cd into the directory
- Do git checkout ch-set-up-eslint-165541393
- Then cd into package.json 
- check dependencies

#### What are the relevant pivotal tracker stories?
[#165541393](https://www.pivotaltracker.com/n/projects/2328795, "set-up eslint")
